Frontend/src/app/components/user-post/user-post.component.html

```diff
@@ -11,12 +11,7 @@
         <span>{{ likes }} Likes</span>
       </div>
       <div class="post-details-row">
-        <a
-          role="button"
-          tabindex="0"
-          (click)="openComments()"
-          (keydown)="onKeyDown($event)"
```

Comment:
could (keydown)="openComments()" work? otherwise, the tabindex will be fine!

Reply:
I was thinking it wouldn't because there will be multiple posts on the screen but I am not sure